code-challenges/fizzBuzzTree/fizz-buzz-tree.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+class Node {
+  constructor(value = null, left = null, right = null) {
+    this.value = value;
+    this.left = left;
+    this.right = right;
+  }
+}
+
+function fizzBuzzTree(treeA) {
+  const rootA = treeA.root;
+  const treeB = new Tree(new Node);
+  const rootB = treeB.root;
+  function fizzBuzz(rootA, rootB) {

```

Comment:
In most cases we do not write a function definition in another function definition. Better to pull it out.